### ISSUE TITLE
Logging in addition to notifications

### ISSUE THREAD

**Issue by jdejager** [DomT4/homebrew-autoupdate#15]
Wouldn't it be useful to add some logging? I've been using autoupdate for a couple of weeks now, but it leaves me wondering what was updated...

I've activated the notifications, but I never see them since the updates usually run when I'm not at my computer.

**Comment by jdejager** [comment]
FYI ... Besides this inconvenience, I love it! Should have been part of brew from the get go IMO.

**Comment by SimonBrazell** [comment]
Would it be possible to have a quick summary in the notification?

**Comment by DomT4** [comment]
> Would it be possible to have a quick summary in the notification?

I’m on holiday this week but I’ll look into it once I’m back. Might be able to get the notification to read the last line of the log or something, perhaps.

**Comment by SimonBrazell** [comment]
Yeah I thought that too so I had a quick look at the logs after I posted this but couldn't find a nice summary of the updates done for the days I looked at.

16 Jan 2020 8:54:34 pm Dominyk Tiller <notifications@github.com>:

**Comment by SimonBrazell** [comment]
Perhaps we could store the output of `brew outdated` before the update is done and then use this in the notification? Great work on this utility btw!

**Comment by RafaelMiquelino** [comment]
> Sorry for the delay here; I watch far too many massively active projects on GitHub & then wonder why I constantly have to dig my way through several thousand notifications.
> 
> There should be logs present in `~/Library/Logs/com.github.domt4.homebrew-autoupdate`?

@DomT4, I issued a PR to include this in the docs.

**Comment by DomT4** [comment]
I'll try and get back into the broader conversation on this handling of notifications in the near future, but I've accepted @RafaelMiquelino's PR to point the log location out more clearly in the meantime.

**Comment by DomT4** [comment]
> Perhaps we could store the output of `brew outdated` before the update is done and then use this in the notification? Great work on this utility btw!

It's not there yet but the new notification system I've started working on contains an update on which taps were updated. Fuller explanation in https://github.com/DomT4/homebrew-autoupdate/issues/25#issuecomment-729088027 and the referenced commits.
